### PR TITLE
upload new recipes

### PR DESCRIPTION
**Pull request by anipenguin**
- Melt Spore
- Transfer Spore/Cluster
- Disable Zero-fee Transfer
- Modify Zero-fee Transfer (post creation)
- added "Modify Zero-fee Transfer" under extra of "Create Spore"

Rearranged order scheme to Create/Transfer/Melt/Data from Spore/Cluster

### CODE REVIEW

**Review by ShookLyngs — CHANGES_REQUESTED** [review]

File: docs/recipes/create-private-cluster.mdx [hunk 1]
```diff
@@ -30,7 +30,7 @@ let { txSkeleton } = await createCluster({
 
 - `data` - The cluster's data, including name and description.
 - `toLock` - The lock script specifies the new cluster's ownership.
-- `fromInfos` - The transaction's sponsors, specifies where to collect capacity from.
+- `fromInfos` - The transaction's sponsor, specifies where to collect capacity from.
```
[the remark this answers is not in the document]
Comment (anipenguin) [reply docs/recipes/create-private-cluster.mdx:33]: can you have multiple addresses to sponsor the transaction?

[the remark this answers is not in the document]
Comment (ShookLyngs) [reply docs/recipes/create-private-cluster.mdx:33]: Yes, `fromInfos` accepts a list of `FromInfo`, you can pass multiple fromInfos like this:

```
fromInfos: [Charlie, Alice],
```

File: docs/recipes/modify-0-fee-transfer.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 8
+---
+
+# Modify Zero-fee Transfer 
+
+Every spore, when created, is fueled with 1 CKBytes as “gas” that covers about `100,000` future transactions. This default setting, referred as the  [Zero-fee Transfer](/basics/spore-101#do-i-need-to-hold-ckb-to-transfer-and-receive-spores) feature enables recipients to transact spores without additional costs.
```
[the remark this answers is not in the document]
Comment (anipenguin) [reply docs/recipes/modify-0-fee-transfer.md:7]: makes sense, i'm drafting a post about the concept of "capacity margin" separately under Get Started

File: docs/recipes/create-private-cluster.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (anipenguin) [reply docs/recipes/create-private-cluster.mdx:33]: okay, was not aware of this, how do they split the transaction fee in practice?

[the remark this answers is not in the document]
Comment (ShookLyngs) [reply docs/recipes/create-private-cluster.mdx:33]: I believe the logic behind this is as follows: 
- `Charlie` pays as much as he can
- If `Charlie` is unable to pay the full fee, `Alice` covers the remaining amount.

File: docs/recipes/disable-0-fee-transfer.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  toLock: OWNER_LOCK,
+  fromInfos: [OWNER_ADDRESS],
+    // highlight-next-line
+  capacityMargin: BI.from(0), // No capacity margin will be added
+});
+```
+
+- `data` - The spore's data, including file data relevant properties.
+- `toLock` - The lock script specifies the spore's ownership.
+- `fromInfos` - The transaction's sponsor, specifies where to collect capacity from.
```
Comment (anipenguin) [inline docs/recipes/disable-0-fee-transfer.md:31]:
```suggestion
- `fromInfos` - The transaction's sponsors, specifies where to collect capacity from.
```

File: docs/recipes/disable-0-fee-transfer.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+import { createSpore } from '@spore-sdk/core';
+import { BI } from '@ckb-lumos/lumos';
+
+let { txSkeleton } = await createSpore({
+  data: {
+    contentType: CONTENT_MIME_TYPE,
+    content: CONTENT_AS_BYTES,
+  },
+  toLock: OWNER_LOCK,
+  fromInfos: [OWNER_ADDRESS],
```
Comment (anipenguin) [inline docs/recipes/disable-0-fee-transfer.md:23]:
```suggestion
  fromInfos: [SPONSOR_ADDRESS],
```

File: docs/recipes/disable-0-fee-transfer.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+import { transferSpore } from '@spore-sdk/core';
+
+let { txSkeleton } = await transferSpore({
+  outPoint: {
+    txHash: '0x76cede56c91f8531df0e3084b3127686c485d08ad8e86ea948417094f3f023f9',
+    index: '0x0',
+  },
+  toLock: RECEIVER_LOCK_SCRIPT,
+    // highlight-next-line
+  fromInfos: [OWNER_ADDRESS],
```
Comment (anipenguin) [inline docs/recipes/disable-0-fee-transfer.md:54]:
```suggestion
  fromInfos: [SPONSOR_ADDRESS],
```

File: docs/recipes/disable-0-fee-transfer.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    // highlight-next-line
+  useCapacityMarginAsFee: false,
+});
+```
+
+- `outPoint` - Specifies the target spore for the transfer, identified by its transaction hash and index.
+- `toLock` - The lock script of the new owner/recipient of the spore.
+- `fromInfos` - The transaction's sponsor, specifies where to collect capacity from.
```
Comment (anipenguin) [inline docs/recipes/disable-0-fee-transfer.md:62]:
```suggestion
- `fromInfos` - The transaction's sponsors, specifies where to collect capacity from.
```

File: docs/recipes/melt-spore.mdx [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+  outPoint: {
+    txHash: '0x76cede56c91f8531df0e3084b3127686c485d08ad8e86ea948417094f3f023f9',
+    index: '0x0',
+  },
+  fromInfos: [OWNER_ADDRESS],
+});
+```
+
+- `outPoint` - Specifies the spore you want to melt, identified by its transaction hash and index.
+- `fromInfos` - The transaction's sponsor, specifies where to collect capacity from.
```
Comment (anipenguin) [inline docs/recipes/melt-spore.mdx:31]:
```suggestion
- `fromInfos` - The transaction's sponsors, specifies where to collect capacity from.
```

File: docs/recipes/melt-spore.mdx [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+You can melt a regular spore using the `meltSpore` API from [spore-sdk](https://docs.spore.pro/resources/spore-sdk):
+```tsx
+import { meltSpore } from '@spore-sdk/core';
+
+let { txSkeleton } = await meltSpore({
+  outPoint: {
+    txHash: '0x76cede56c91f8531df0e3084b3127686c485d08ad8e86ea948417094f3f023f9',
+    index: '0x0',
+  },
+  fromInfos: [OWNER_ADDRESS],
```
Comment (anipenguin) [inline docs/recipes/melt-spore.mdx:26]:
```suggestion
  fromInfos: [SPONSOR_ADDRESS],
```

File: docs/recipes/modify-0-fee-transfer.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+If you want to modify this feature by adding additional CKBytes, or if the spore was created without the capacity margin, you can utilize the `transferSpore` API to "refuel":
+
+```tsx
+import { transferSpore } from '@spore-sdk/core';
+import { BI } from '@ckb-lumos/lumos';
+
+let { txSkeleton } = await transferSpore({
+  outPoint: SPORE_OUTPOINT,
+  toLock: OWNER_LOCK,
+  fromInfos: [OWNER_ADDRESS],
```
Comment (anipenguin) [inline docs/recipes/modify-0-fee-transfer.md:20]:
```suggestion
  fromInfos: [SPONSOR_ADDRESS],
```

File: docs/recipes/modify-0-fee-transfer.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+  // highlight-start
+  capacityMargin: BI.from(1_0000_0000), // Add 1 CKB as margin, default is 0
+  useCapacityMarginAsFee: false, // Disable the feature
+  // highlight-end
+});
+```
+
+- `outPoint` - Specifies the target spore for configuration, identified by its transaction hash and index.
+- `toLock` - The lock script of the new owner/recipient of the spore (which remains with the original owner in this case).
+- `fromInfos` - The transaction's sponsor, specifies where to collect capacity from.
```
Comment (anipenguin) [inline docs/recipes/modify-0-fee-transfer.md:30]:
```suggestion
- `fromInfos` - The transaction's sponsors, specifies where to collect capacity from.
```

**Review by anipenguin — COMMENTED** [review]
[OWNER_ADDRESS] -> [SPONSOR_ADDRESS] to avoid confusion

File: docs/recipes/create-spore.mdx [hunk 11]
```diff
@@ -44,6 +44,22 @@ Here's a code example for how to create a spore with [`CKB default lock`](https:
   className="margin-top--sm"
 />
 
+### Modify the Zero-fee Transfer feature
+
+The Zero-fee Transfer feature is enabled by reserving a small amount of CKBytes as fee for future transactions. When creating a spore, the default is to allocate 1 CKB (100,000,000 shannons) as capacity margin. To adjust the margin amount for the new spore, you can set the `capacityMargin` prop, for example, to 2 CKB like this:
```
[the remark this answers is not in the document]
Comment (anipenguin) [reply docs/recipes/create-spore.mdx]: updated to "The Zero-fee Transfer feature enables recipients to transact spores without additional costs by reserving a tiny amount of CKBytes as fee for future transactions. When creating a spore, the default is to allocate 1 CKB (100,000,000 shannons) as capacity margin to cover about 100,000 future transaction."

**Review by anipenguin — COMMENTED** [review]
make modifications accordingly

**Review by anipenguin — COMMENTED** [review]
made changes accordingly

**Review by ShookLyngs — APPROVED** [review]

**Pull request merged**